flutter/macos/sentry_flutter.podspec

```diff
@@ -0,0 +1,22 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint sentry_flutter.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'sentry_flutter'
+  s.version          = '0.0.1'
```

Comment:
Do we need to account for bumping this during our release process?

Reply:
No, this never gets published to the CocoaPods registry. It is also just symlinked into the correct folder, so even if we would do that, it wouldn't matter.